src/Quartz/Impl/StdSchedulerFactory.cs

```diff
@@ -138,7 +138,11 @@ public class StdSchedulerFactory : ISchedulerFactory
 
         private static readonly ILog log = LogProvider.GetLogger(typeof(StdSchedulerFactory));
 
-        private string SchedulerName => cfg.GetStringProperty(PropertySchedulerInstanceName, "QuartzScheduler");
+        private string SchedulerName
```

Comment:
isn't this basically the very same code, expression bodied members translate to get/set?

Reply:
No it isn't, I thought the same as you but it looks like that the expression body way generates a backing field with a null check. Just try it out and you will see that the expression body works different then the property with only a getter.